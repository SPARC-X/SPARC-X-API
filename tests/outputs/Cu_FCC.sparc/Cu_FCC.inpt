
LATVEC_SCALE: 5.416914 5.416914 5.416914   
LATVEC:                       
  1.00 0.00 0.00
  0.00 1.00 0.00
  0.00 0.00 1.00
BC: P P P                    


EXCHANGE_CORRELATION: GGA_PBE  


ELEC_TEMP_TYPE: Fermi-Dirac   
ELEC_TEMP: 300                


MESH_SPACING: 0.22      

KPOINT_GRID: 1 1 1           



PRECOND_KERKER_THRESH: 0.0   
MIXING_PARAMETER: 1.0         
TOL_SCF: 1e-4                 



CALC_STRESS: 0
PRINT_ATOMS: 1
PRINT_VELS: 0
PRINT_FORCES: 0
PRINT_EIGEN: 0
PRINT_DENSITY: 0
