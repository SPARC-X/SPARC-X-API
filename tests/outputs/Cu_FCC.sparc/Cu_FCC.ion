#=========================
# format of ion file
#=========================
# ATOM_TYPE: <atom type name>
# N_TYPE_ATOM: <num of atoms of this type>
# COORD:
# <xcoord> <ycoord> <zcoord>
# ...
# RELAX:
# <xrelax> <yrelax> <zrelax>
# ...

ATOM_TYPE: Cu                 # atom type
ATOMIC_MASS: 63.546           # atomic mass (amu)
PSEUDO_POT: ../../../psps/29_Cu_19_1.7_1.9_pbe_n_v1.0.psp8  # pseudopotential file
N_TYPE_ATOM: 4                # number of atoms of this type
# COORD:                      # Cartesian coordinates (au)
COORD_FRAC:                   # fractional coordinates (in lattice vector basis)
   0.0 0.0 0.0
   0.0 0.5 0.5
   0.5 0.0 0.5
   0.5 0.5 0.0
