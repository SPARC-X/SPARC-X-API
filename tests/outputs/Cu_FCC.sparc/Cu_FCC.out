***************************************************************************
*                       SPARC (version Feb 03, 2023)                      *
*   Copyright (c) 2020 Material Physics & Mechanics Group, Georgia Tech   *
*           Distributed under GNU General Public License 3 (GPL)          *
*                   Start time: Sun Feb  5 13:19:37 2023                  *
***************************************************************************
                           Input parameters                                
***************************************************************************
LATVEC_SCALE: 5.416914 5.416914 5.416914 
LATVEC:
1 0 0 
0 1 0 
0 0 1 
FD_GRID: 25 25 25
FD_ORDER: 12
BC: P P P
KPOINT_GRID: 1 1 1
KPOINT_SHIFT: 0 0 0
SPIN_TYP: 0
ELEC_TEMP_TYPE: Fermi-Dirac
SMEARING: 0.000950043469
EXCHANGE_CORRELATION: GGA_PBE
NSTATES: 50
CHEB_DEGREE: 33
CHEFSI_BOUND_FLAG: 0
CALC_STRESS: 0
CALC_PRES: 0
MAXIT_SCF: 100
MINIT_SCF: 2
MAXIT_POISSON: 3000
TOL_SCF: 1.00E-04
POISSON_SOLVER: AAR
TOL_POISSON: 1.00E-06
TOL_LANCZOS: 1.00E-02
TOL_PSEUDOCHARGE: 1.00E-07
MIXING_VARIABLE: density
MIXING_PRECOND: kerker
TOL_PRECOND: 4.69E-05
PRECOND_KERKER_KTF: 1
PRECOND_KERKER_THRESH: 0
MIXING_PARAMETER: 1
MIXING_HISTORY: 7
PULAY_FREQUENCY: 1
PULAY_RESTART: 0
REFERENCE_CUTOFF: 0.5
RHO_TRIGGER: 4
FIX_RAND: 0
VERBOSITY: 1
PRINT_FORCES: 0
PRINT_ATOMS: 1
PRINT_EIGEN: 0
PRINT_DENSITY: 0
PRINT_ENERGY_DENSITY: 0
OUTPUT_FILE: Cu_FCC/temp_run/Cu_FCC
***************************************************************************
                                Cell                                       
***************************************************************************
Lattice vectors (Bohr):
5.416914000000000 0.000000000000000 0.000000000000000 
0.000000000000000 5.416914000000000 0.000000000000000 
0.000000000000000 0.000000000000000 5.416914000000000 
Volume: 1.5894827611E+02 (Bohr^3)
***************************************************************************
                           Parallelization                                 
***************************************************************************
NP_SPIN_PARAL: 1
NP_KPOINT_PARAL: 1
NP_BAND_PARAL: 6
NP_DOMAIN_PARAL: 2 2 2
NP_DOMAIN_PHI_PARAL: 3 4 4
EIG_SERIAL_MAXNS: 1500
***************************************************************************
                             Initialization                                
***************************************************************************
Number of processors               :  48
Mesh spacing                       :  0.216677 (Bohr)
Number of symmetry adapted k-points:  1
Output printed to                  :  Cu_FCC/temp_run/Cu_FCC.out
Total number of atom types         :  1
Total number of atoms              :  4
Total number of electrons          :  76
Atom type 1  (valence electrons)   :  Cu 19
Pseudopotential                    :  ../psps/29_Cu_19_1.7_1.9_pbe_n_v1.0.psp8
Pseudocharge radii of atom type 1  :  6.93 6.93 6.93 (x, y, z dir)
Number of atoms of type 1          :  4
Estimated total memory usage       :  39.10 MB
Estimated memory per processor     :  834.15 kB
===================================================================
                    Self Consistent Field (SCF#1)                     
===================================================================
Iteration     Free Energy (Ha/atom)   SCF Error        Timing (sec)
1            -1.8219384019E+02        1.309E-02        0.156
2            -1.8219366364E+02        2.830E-03        0.030
3            -1.8219480403E+02        1.155E-02        0.031
4            -1.8219379510E+02        7.477E-03        0.030
5            -1.8219366053E+02        2.475E-03        0.030
6            -1.8219365180E+02        1.748E-03        0.034
7            -1.8219364739E+02        1.178E-03        0.030
8            -1.8219364419E+02        1.289E-04        0.030
9            -1.8219364489E+02        5.124E-05        0.029
Total number of SCF: 9     
====================================================================
                    Energy and force calculation                    
====================================================================
Free energy per atom               : -1.8219364489E+02 (Ha/atom)
Total free energy                  : -7.2877457955E+02 (Ha)
Band structure energy              : -5.1914421254E+01 (Ha)
Exchange correlation energy        : -8.8853744679E+01 (Ha)
Self and correction energy         : -7.4614360114E+02 (Ha)
-Entropy*kb*T                      : -3.6282748788E-03 (Ha)
Fermi level                        :  7.2899024066E-01 (Ha)
RMS force                          :  9.6572290763E-06 (Ha/Bohr)
Maximum force                      :  1.1835919062E-05 (Ha/Bohr)
Time for force calculation         :  0.051 (sec)
***************************************************************************
                               Timing info                                 
***************************************************************************
Total walltime                     :  0.998 sec
___________________________________________________________________________

***************************************************************************
*             Material Physics & Mechanics Group, Georgia Tech            *
*                       PI: Phanish Suryanarayana                         *
*               List of contributors: See the documentation               *
*         Citation: See README.md or the documentation for details        *
*  Acknowledgements: U.S. DOE SC (DE-SC0019410), U.S. DOE NNSA (ASC)      *
*      {Preliminary developments: U.S. NSF (1333500,1663244,1553212)}     *
***************************************************************************
                                                                           
